### DISCUSSION TITLE
Defaults for Self Hosted Khoj

### DISCUSSION THREAD

**Discussion by KShivum** [khoj-ai/khoj#871] [Q&A]
Hi, so I'm trying out Khoj and I set it up to run, but it really feels like the documentation leaves a lot to be desired. Especially since after you set up, you cannot use the program since you need to set up an Agent which has items that are not explained, for instance, what are tools for an agent? It is a required field and I can't find any examples of it. Having some default items for many of the things in the program would help especially if it isn't documented

**Comment by rehandaphedar** [comment]
I will be trying a bare metal install sometime soon, I'll see if I can add some documentation based on what I learn. In the meanwhile, the Docker install is usable out of the box, you could use it as a reference.

**Reply by debanjum** [reply]
Yeah, we've done a bunch of changes to make self-hosting require less work to get started (e.g  remove need to configure khoj on the admin panel to get started). 

I've also added [documentation](https://docs.khoj.dev/advanced/admin) about the more important knobs on the admin panel.

Hope is to allow folks to just `docker-compose up` to get started, and explore all the customizability as an optional next step.

Feedback like this is super helpful to gauge how much work is required to get there!
